ospkg/create_test.go

```diff
@@ -1,9 +1,13 @@
-package ospkg
+package ospkg_test
```

Comment:
Why is it a different package?

Reply:
Can't give a clever explanation other than the new linter wanted it that way and the IDE also recommends it now with the replaced linter. The compiler also does not complain so I guess it was meant to place test code under "packagename_test" in the same folder even if that string isn't identical to "packagename"

Reply:
In `pkgname_test`, can you still access unexpected things from `pkgname`?

Reply:
https://stackoverflow.com/questions/19998250/proper-package-naming-for-testing-with-the-go-language

I think we should disable this test. From my point of view, white-box testing is worth it, and we should be able to write the tests in the same package. As pointed above, there is nothing wrong with it.

Reply:
Yes I agree, checked the docs and they also don't explicitly recommend using the _test package name. I reverted this change and disabled the linting test that checks those.